public/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf8">
+        <title>Mock Up Open Table Reservation Module</title>
```

Comment:
@LittleJennie 
@cminho921 

Would change this document title as well!

Reply:
Thank you for point this out! HTML title will be updated in the next PR.